lib/graphiti/sideload/polymorphic_belongs_to.rb

```diff
@@ -15,9 +15,14 @@ def method_missing(name, *args, &blk)
   class Grouper
     attr_reader :field_name
 
-    def initialize(field_name)
+    def initialize(field_name, opts={})
       @field_name = field_name
       @groups = []
+      @ignore_groups = Array(opts[:ignore_groups])
+    end
+
+    def ignore_group?(group_name)
+      @ignore_groups.include?(group_name.to_sym)
```

Comment:
If casting the group name `to_sym` on check, I would do the same to the ignore list when assigning in the initializer.

Reply:
done